root/programs/C#/Frameworks/Tools/DPQuery_Tool/Form1.cs

```diff
@@ -584,7 +626,7 @@ private void cmbDataProvider_SelectedIndexChanged(object sender, EventArgs e)
                     this.nudNumOfBind.Enabled = true;
 
                     // 状態
-                    ((ToolStripStatusLabel)this.statBar.Items[0]).Text = this.RM_GetString("STATUS_OLE_CREATED");
+                    ((ToolStripStatusLabel)this.statBar.Items[0]).Text = string.Format(this.RM_GetString("STATUS_DATA_PROVIDER_SELECTED"), Literal.DAP_OLE);
 
                 }
                 if (this.cmbDataProvider.SelectedItem.ToString() == Literal.DAP_ODB)
```

Comment:
@yusukemaegawa 

This pointed out is another matter.
Therefore, this will register as another issue.